collector/analyzer/tcpconnectanalyzer/analyzer.go

```diff
@@ -182,6 +182,8 @@ func (a *TcpConnectAnalyzer) generateDataGroup(connectStats *internal.Connection
 	// Only record the connection's duration when it is successfully established
 	if connectStats.StateMachine.GetCurrentState() == internal.Success {
 		metrics = append(metrics, model.NewIntMetric(constnames.TcpConnectDurationMetric, connectStats.GetConnectDuration()))
+	} else {
+		metrics = append(metrics, model.NewIntMetric(constnames.TcpConnectDurationMetric, 0))
```

Comment:
Such changes can seriously affect the average connection time. How about setting this value to an impossible value and adding something special logic when aggregating?

Reply:
This is a serious problem I didn't notice before. Adding special logic when aggregating is not a good idea as aggregating processor shouldn't know too much about the data itself. There isn't an appropriate method that arose in my mind to fix this issue now. I will close this PR temporarily.